app/controllers/api/account.php

```diff
@@ -2369,15 +2370,13 @@
             throw new Exception(Exception::USER_SESSION_NOT_FOUND);
         }
 
-        $jwt = new JWT(System::getEnv('_APP_OPENSSL_KEY_V1'), 'HS256', 900, 10); // Instantiate with key, algo, maxAge and leeway.
+        $jwt = new JWT(System::getEnv('_APP_OPENSSL_KEY_V1'), 'HS256', 3600, 0);
 
         $response
             ->setStatusCode(Response::STATUS_CODE_CREATED)
             ->dynamic(new Document(['jwt' => $jwt->encode([
-                // 'uid'    => 1,
-                // 'aud'    => 'http://site.com',
-                // 'scopes' => ['user'],
-                // 'iss'    => 'http://api.mysite.com',
+                'iat' => \time(),
+                'exp' => \intval((new \DateTime())->add(new \DateInterval('PT900S'))->format('U')),
```

Comment:
the maxAge param when initializing the JWT is used for the exp. Why do we need to manually add these things in?

Reply:
Used maxAge instead